static/app/components/core/input.tsx

```diff
@@ -99,5 +100,5 @@ export const Input = styled(
   ),
   {shouldForwardProp: prop => typeof prop === 'string' && isPropValid(prop)}
 )`
-  ${inputStyles}
+  ${p => (p.theme.isChonk ? chonkInputStyles(p as any) : inputStyles(p))}
```

Comment:
this isn’t great, but it works for the input as we need to:

- expose as `styled()` component because we use `.withComponent` now
- styling needs to be applied _before_ we render the `Input` component as `size` should get filtered out
- there is no props mapping so `withChonk` seems overkill
- `withChonk` doesn’t do `forwardRef` right now